src/content/docs/webdev-hosting/vercel/vercel-case-study.md

```diff
@@ -0,0 +1,130 @@
+| 2022     | Vercel acquired Splitbee                                                               |
+| 2023     | Vercel Postgres was introduced, a serverless PostgresAQL database integrated in Vercel |
+|          | Vercel introduced v0 AI driver creation of code and design                             |
+
+## Main Features
+
+Some of Vercel's features:
+
+- **Infrastructure and security:** Vercel utilizes an Edge Network for intelligent traffic routing near users, offering dynamic scalability while ensuring default security through isolated request handling and global content replication for stability.
+- **Instant deployment:** Vercel enables instant deployment by connecting directly to GIT repositories. When developers push new code trough GIT, Vercel automatically detects changes and deploys the updated version without manual intervention.
+- **The native Next.js platform:** Vercel's platform is made by the creators of Next.js, so Next.js applications will have multiple advantages when deployed to Vercel compared to other similar hosting solutions.
+- **Vercel PostgreSQL:** is a serverless SQL database that is integrated in Vercel Functions. It enables you to create secure PostgreSQL databases that are easily scalable.
+- **Vercel KV:** is a durable Redis database that enabled users to store and retrieve JSON data.
+- **Vercel Blob ( Beta ):** allows users to serve and upload files trough a global network with unique and unguessable URLs.
+- **Vercel Edge config:** is a global data store that enables experimentation with feature flags, A/B testing, critical redirects, and more. The vast majority of reads from the Edge Config will complete within 15ms to as low as 0ms in some scenarios.
+
+Feature table:
+
+| **Feature**                 | **Description**                                         |
+|-----------------------------|---------------------------------------------------------|
+| Infrastructure and security | Optimizes performance via global server network         |
+| Instant deployment          | Automatically deploys changes from Git repos            |
+| The native Next.js platform | Many big advantages with Next.js applications           |
+| Vercel PostgreSQL           | Serverless SQL database integrated in Vercel            |
+| Vercel KV                   | Serverless Redis database using JSON data               |
+| Vercel Blob ( Beta )        | Serve and upload files with unique and unguessable URLs |
+| Vercel Edge config          | Global data store with low latency and many features    |
+
+## Market Comparison
+
+Compared to other hosting platforms, Vercel stands out for its seamless integration with version control systems, automated deployment, and its focus on simplicity.
+
+Next.js is a very popular react framework with big advantages over vanilla react, Vercel has many big advantages with Next.js applications that the competition does not have because the developers behind Vercel is the same as Next.js.
```

Comment:
```suggestion
Next.js is a very popular react framework with big advantages over vanilla React, Vercel has many big advantages with Next.js applications that the competition does not have because the developers behind Vercel are the same as Next.js.
```